### PR TITLE
Updated sentry_set_commits

### LINKED ISSUE #179: Requesting to support `--ignore-missing` in sentry_set_commits
Currently, sentry_set_commits does not offer a way to manage --ignore-missing through passing arguments. But it would be really helpful if we can use the same feature in fastlane plugin which is already built for the CLI.

### PR DESCRIPTION
**Pull request by samitha9125**
New feature

1. Updated sentry_set_commits to support --ignore missing.

This PR resolves #179.

### CODE REVIEW

**Review by philipphofmann — COMMENTED** [review]
Thanks, @samitha9125, for doing this. Please update the ReadM here https://github.com/getsentry/sentry-fastlane-plugin/blob/19a59a4305a44a83868abc15cc4c64d10d353c48/README.md?plain=1#L170-L178

also please add a test for this new functionality to https://github.com/getsentry/sentry-fastlane-plugin/blob/master/spec/sentry_set_commits_spec.rb

**Comment by samitha9125** [comment]
@philipphofmann Updated the PR with tests and Readme updates.

File: README.md
Comment (philipphofmann) [inline README.md:178]:
```suggestion
  ignore_previous_commits: false # Optional boolean value: When the flag is set and the previous release commit was not found in the repository, will create a release with the default commits count (or the one specified with `--initial-depth`) instead of failing the command.
```

**Comment by samitha9125** [comment]
@philipphofmann Thank you for valuable suggestions. I did necessary changes.

**Comment by samitha9125** [comment]
@philipphofmann Can you please approve this if there are nothing else to improve?

File: lib/fastlane/plugin/sentry/actions/sentry_set_commits.rb
Comment (philipphofmann) [inline lib/fastlane/plugin/sentry/actions/sentry_set_commits.rb:68]:
```suggestion
                                       description: "When enabled, if the previous release commit was not found in the repository, will create a release with the default commits count (or the one specified with `--initial-depth`) instead of failing the command.",
```

**Review by philipphofmann — COMMENTED** [review]
Last missing bit. Thanks.

**Comment by samitha9125** [comment]
@philipphofmann Done :)

**Review by philipphofmann — APPROVED** [review]
LGTM

**Comment by philipphofmann** [comment]
Please fix CI @samitha9125.

**Comment by samitha9125** [comment]
@philipphofmann, Sorry, my bad. Now it is fixed.

**Comment by samitha9125** [comment]
@philipphofmann Any possibility of running these tests locally? I mean the command? So I can check it completely and update the PR.

**Comment by philipphofmann** [comment]
Of course, it should be possible to run that locally. If you use VSCode and setup ruby properly, it shouldn't be a problem.

**Comment by samitha9125** [comment]
@philipphofmann Thank mate. Could fix all issued related to the new param. However, I could not resolve one error related to danger ci. Can you check now?

**Comment by philipphofmann** [comment]
You need to add an entry to the Changelog @samitha9125 under unreleased/features with linking to this PR.
So add 
`set_commits` supports `--ignore missing` (#180)

**Comment by samitha9125** [comment]
@philipphofmann Done :) Once this is merged, is there any planned time to release this feature?

**Review by philipphofmann — APPROVED** [review]
LGTM

File: CHANGELOG.md
Comment (philipphofmann) [reply CHANGELOG.md:11]: Ah it actually was. I don't understand why danger complained.

**Comment by philipphofmann** [comment]
> @philipphofmann Done :) Once this is merged, is there any planned time to release this feature?

Release is out @samitha9125.

**Pull request merged**
